### DISCUSSION TITLE
Trending section used to be a mix of movies and shows, now only showing Asian actors

### DISCUSSION THREAD

**Discussion by fcassata** [seerr-team/seerr#1034] [Q&A] [answered]
Is there a way to:

1. Filter out Asian content entirely?
2. Make the trending section show a mix of trending movies and shows?

I'm on 2.0.1, but before upgrading the trending section had only been showing series for a few days, no movies.

**Comment by gauthier-th** [comment]
This is not an issue from 2.0.1. This is an issue related to TMDB, which returns incorrect data.

**Reply by fcassata** [reply]
Is there a way to filter the data to show only series or only movies?

**Reply by fallenbagel** [reply, marked as answer]
> Is there a way to filter the data to show only series or only movies?

It's supposed to work like that. But that endpoint from tmdb side is broken. There's nothing we can do till they fix it. Which they have slowly been apparently they fixed it on their website now and we jusy have to wait till it propagates to the api

**Reply by fcassata** [reply]
Ahh gotcha! Thanks for the info.

**Reply by fallenbagel** [reply]
https://www.themoviedb.org/talk/6711c96f097c3d7760f87c6a
